lang/cs/GrpcCodeGeneration.targets

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<!-- AVRO CODE GERNATION TARGETS FOR MSBUILD -->
```

Comment:
Do we still need Avro targets?

Reply:
No incorrect comment will modify.

Reply:
Also, does gRPC come with a plugin for this already we can just use?

Reply:
I am currently using the gRPC nuget tools and core nuget packages produced by the gRPC team.

Reply:
Their examples use a batch file to access the tools in the packages directory. I did this with C# code in an msbuild task.